optimer/views.py

```diff
@@ -0,0 +1,82 @@
+
+
+from django.http import HttpResponseRedirect
+from django.template import RequestContext
+from django.shortcuts import render_to_response
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.decorators import permission_required
+from django.contrib.auth.decorators import user_passes_test
+from django.utils import timezone
+
+from util import check_if_user_has_permission
+from authentication.managers import AuthServicesInfoManager
+from eveonline.managers import EveManager
+from form import opForm
+from models import optimer
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+def optimer_util_test(user):
+    return check_if_user_has_permission(user, 'member') or check_if_user_has_permission(user, 'blue_member')
+
+
+@login_required
+@user_passes_test(optimer_util_test)
+@permission_required('auth.optimer_view')
+def optimer_view(request):
+    logger.info("optimer_view called by user %s" % request.user)
+    optimer_list = optimer.objects.all()
+    render_items = {'optimer': optimer.objects.all(),}
+
+    return render_to_response('registered/operationmanagement.html', render_items, context_instance=RequestContext(request))
+
+
+@login_required
+@permission_required('auth.optimer_management')
+def add_optimer_view(request):
+    logger.info("add_optimer_view called by user %s" % request.user)
+    if request.method == 'POST':
+        form = opForm(request.POST)
+	logger.info("Request type POST contains form valid: %s" % form.is_valid())
```

Comment:
haha you're only getting away with it because you're not on ubuntu - you have no idea how many times I have to test import a new file before I sort my indentations out